extension/src/popup/components/sendPayment/SendAmount/SendType/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Field, Form, Formik } from "formik";
+import { Button, Icon } from "@stellar/design-system";
+
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { PopupWrapper } from "popup/basics/PopupWrapper";
+import { AutoSaveFields } from "popup/components/AutoSave";
+import {
+  saveDestinationAsset,
+  transactionDataSelector,
+} from "popup/ducks/transactionSubmission";
+
+import "./styles.scss";
+
+interface RadioCheckProps {
+  name: string;
+  title: string;
+  subtext: string;
+  value: string;
+}
+
+const RadioCheck = ({ name, title, subtext, value }: RadioCheckProps) => (
+  <>
+    <label className="SendType--label SendType--radio-label">
+      <div className="SendType__title">
+        {/* TODO - tooltip copy */}
+        {title}
+        {subtext && <span className="SendType__title__subtext">{subtext}</span>}
+      </div>
+      <Field
+        className="SendType--radio-field"
+        name={name}
+        type="radio"
+        value={value}
+      />
+      <div className="SendType--radio-check">
+        <Icon.Check />
+      </div>
+    </label>
+  </>
+);
+
+export const SendType = () => {
+  const dispatch = useDispatch();
+  const { destinationAsset } = useSelector(transactionDataSelector);
+  return (
+    <PopupWrapper>
+      <div
+        className="SendType__btn-exit"
+        onClick={() => navigateTo(ROUTES.sendPaymentAmount)}
+      >
+        <Icon.X />
+      </div>
+      <div className="SendPayment__header">SendType</div>
+      <Formik
+        initialValues={{ isPathPayment: String(destinationAsset !== "") }}
+        onSubmit={(values) => {
+          dispatch(
+            saveDestinationAsset(
+              values.isPathPayment === "true" ? "native" : "",
+            ),
+          );
+        }}
+      >
+        <Form className="SendType__form">
+          <AutoSaveFields />
+          <RadioCheck
+            name="isPathPayment"
+            title="Same source and destination asset"
+            value="false"
```

Comment:
til react radio inputs dont like boolean values

Reply:
Yup! Kinda weird, but they want you to assign a string to it, similar to an `input`

Since `value` is going to be a string (and not a boolean), I think it'd be best not to use `isPathPayment` with `true` or `false` strings. It's a little misleading since you're expecting a boolean, but you're just matching a string.

I think it might be good to just treat it how we would treat it as any other string value. 

Something like:

```
enum PaymentTypes {
  REGULAR_PAYMENT = "REGULAR_PAYMENT", // better naming lol
  PATH_PAYMENT = "PATH_PAYMENT"
}

<RadioCheck
            name="paymentType"
            title="Same source and destination asset"
            value="PaymentTypes.REGULAR_PAYMENT"
```

for ex: https://github.com/stellar/freighter/blob/release/2.0.0/extension/src/popup/views/Preferences/index.tsx#L101

Reply:
ya  I tried making it "simple" with the boolean and React said nope lol, makes sense for enum
https://github.com/stellar/freighter/pull/411/commits/97ff8160efae70796e03842de89807b8f1a8677b
(and added the continue btn disable if loading)